GoogleForCreators/web-stories-wp
# Add Moveable for Text Presets

## Context
Allows inserting text presets by drag and drop.


## Summary



## Relevant Technical Choices



## To-do




## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Open the Text elements panel
2. Try dragging the presets (Title 1, Paragraph, etc.) onto the Page
3. Verify the texts are inserted as expected
4. Verify it's still possible to insert texts via clicking as well
5. Verify it's still possible to insert Text Presets via keyboard as well


## Checklist




---



Fixes #8359
